### PR TITLE
Implement StudentTabs component logic

### PR DESCRIPTION
**Pull request by Silent777**
Create Mentor app
Create MentorStudent model
Create initial migration file
Create view file with GET and POST methods
Create service for the StudentsTabs componet with GET and POST methods
Create helper.js file with objects compare functions
Update StudentsTabs UI component

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/mentor/urls.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url
+from .views import MentorView
+
+urlpatterns = [
+    url(r'^students/$', MentorView.as_view())
```
Comment (Silent777) [inline eventually/mentor/urls.py]: add name

File: eventually/mentor/views.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+class MentorView(View):
+    """Mentor view handles GET, POST, PUT, DELETE requests."""
+
+    def get(self, request):
+        """
+        Method that handles GET request.
+        """
+        mentor = CustomUser.get_by_id(request.user.id)
+        mentor_topics = mentor.topic_set.all()
```
Comment (Silent777) [inline eventually/mentor/views.py]: topics_id = [record.id for record in mentor.topic_set.all()]

File: eventually/mentor/views.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        mentor_topics = [record.id for record in mentor_topics]
+
+        all_students = MentorStudent.objects.exclude(mentor_id=request.user.id)
```
Comment (Silent777) [inline eventually/mentor/views.py]: move to model method

File: eventually/mentor/views.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        all_students = all_students.exclude(mentor_id=None)
+
+        my_students = MentorStudent.objects.filter(mentor_id=request.user.id)
```
Comment (Silent777) [inline eventually/mentor/views.py:32]: move to model method

File: eventually/mentor/views.py [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+        available_students = MentorStudent.objects.filter(mentor_id=None)
```
Comment (Silent777) [inline eventually/mentor/views.py:34]: move to model method

File: eventually/mentor/views.py [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+        # is_done = True if request.GET.get('topic', "") == 'true' else False
+
+        if request.GET.get('topic', None):
```
Comment (Silent777) [inline eventually/mentor/views.py:38]: :-1:

File: eventually/static/src/components/studentsTabsList/StudentsTabsList.js [hunk 7]
```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-import {Tabs, Tab} from 'material-ui/Tabs';
+import { Tabs, Tab } from 'material-ui/Tabs';
```
Comment (Silent777) [inline eventually/static/src/components/studentsTabsList/StudentsTabsList.js]: fix

File: eventually/static/src/components/studentsTabsList/StudentsTabsList.js [hunk 8]
```diff
@@ -12,51 +14,130 @@ export default class StudentsTabsList extends React.Component {
...
+
     handleChange = (value) => {
         this.setState({
             slideIndex: value
         });
     };
 
-    handleModalOpen = () => {
-        this.setState({isModalOpen: true});
+    handleModalOpen = (studentId) => {
```
Comment (Silent777) [inline eventually/static/src/components/studentsTabsList/StudentsTabsList.js]: ` = studentId =>`

File: eventually/static/src/components/studentsTabsList/StudentsTabsList.js [hunk 9]
```diff
@@ -12,51 +14,130 @@ export default class StudentsTabsList extends React.Component {
...
+                    studentsList: {
+                        myStudents: data.my_students,
+                        allStudents: data.all_students,
+                        availableStudents: data.available_students
+                    }
+                });
+            });
+    };
+
     handleChange = (value) => {
```
Comment (Silent777) [inline eventually/static/src/components/studentsTabsList/StudentsTabsList.js]: `= value =>`

**Review by redmyt — APPROVED** [review]

**Pull request merged**